linera-service/src/linera/main.rs

```diff
@@ -1290,6 +1290,10 @@ fn main() -> anyhow::Result<()> {
         builder
     };
 
+    if let Some(blocking_threads) = options.tokio_blocking_threads {
+        runtime.max_blocking_threads(blocking_threads);
+    }
```

Comment:
Please handle the option in the same for `linera`, `linera-proxy`, and `linera-server`.